server/src/main/java/org/opensearch/action/support/broadcast/BroadcastRequest.java

```diff
@@ -123,6 +126,15 @@ public boolean includeDataStreams() {
     public void writeTo(StreamOutput out) throws IOException {
         super.writeTo(out);
         out.writeStringArrayNullable(indices);
+        out.writeBoolean(shouldCancelOnTimeout);
```

Comment:
Any new field in writeTo or StreamInput reader in constructor has to have version check

Reply:
only added one new shouldCancelOnTimeout bollean.